x-pack/plugins/search_playground/public/components/message_list/assistant_message.tsx

```diff
@@ -0,0 +1,121 @@
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import type { AIMessage as AIMessageType } from '../../types';
+
+import { CopyActionButton } from './copy_action_button';
+import { CitationsTable } from './citations_table';
+import { RetrievalDocsFlyout } from './retrieval_docs_flyout';
+
+type AssistantMessageProps = Pick<
+  AIMessageType,
+  'content' | 'createdAt' | 'citations' | 'retrievalDocs'
+>;
+
+export const AssistantMessage: React.FC<AssistantMessageProps> = ({
+  content,
+  createdAt,
+  citations,
+  retrievalDocs,
+}) => {
+  const [isDocsFlyoutOpen, setIsDocsFlyoutOpen] = useState(false);
+
+  return (
+    <EuiComment
+      username={i18n.translate('xpack.searchPlayground.chat.message.assistant.username', {
+        defaultMessage: 'AI',
+      })}
+      event={i18n.translate('xpack.searchPlayground.chat.message.assistant.event', {
+        defaultMessage: 'responded',
+      })}
+      timestamp={
+        createdAt &&
+        i18n.translate('xpack.searchPlayground.chat.message.assistant.createdAt', {
+          defaultMessage: 'on {date}',
+          values: {
+            date: moment(createdAt).format('MMM DD, YYYY'),
+          },
+        })
+      }
+      timelineAvatar="compute"
+      timelineAvatarAriaLabel={i18n.translate(
+        'xpack.searchPlayground.chat.message.assistant.avatarAriaLabel',
+        {
+          defaultMessage: 'AI',
+        }
+      )}
+      actions={
+        <CopyActionButton
+          copyText={String(content)}
+          ariaLabel={i18n.translate('xpack.searchPlayground.chat.message.assistant.copyLabel', {
+            defaultMessage: 'Copy assistant message',
+          })}
+        />
+      }
+    >
+      <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+        <EuiTitle size="xs">
+          <p>
+            {i18n.translate('xpack.searchPlayground.chat.message.assistant.title', {
+              defaultMessage: 'Summary',
+            })}
+          </p>
+        </EuiTitle>
+        {!!retrievalDocs?.length && (
+          <>
+            <EuiButtonEmpty onClick={() => setIsDocsFlyoutOpen(true)}>
+              {i18n.translate('xpack.searchPlayground.chat.message.assistant.retrievalDocButton', {
+                defaultMessage:
+                  '{count} {count, plural, one {document} other {documents}} retrieved',
+                values: { count: retrievalDocs.length },
+              })}
+            </EuiButtonEmpty>
+
+            {isDocsFlyoutOpen && (
+              <RetrievalDocsFlyout
+                onClose={() => setIsDocsFlyoutOpen(false)}
+                retrievalDocs={retrievalDocs}
+              />
+            )}
+          </>
+        )}
+      </EuiFlexGroup>
+      <EuiSpacer size="m" />
+      <EuiText size="s">
+        <p>{content}</p>
+      </EuiText>
+      {!!citations?.length && (
+        <>
+          <EuiSpacer size="l" />
+          <EuiTitle size="xs">
+            <p>
```

Comment:
nitpick: should this be a `h4` since its in a `EuiTitle` & using `xs` size?

Reply:
Yup